src/pages/home/report/ReportActionCompose/ComposerWithSuggestions.js

```diff
@@ -330,23 +331,7 @@ function ComposerWithSuggestions({
      * @memberof ReportActionCompose
      */
     const focus = useCallback((shouldDelay = false) => {
-        // There could be other animations running while we trigger manual focus.
-        // This prevents focus from making those animations janky.
-        InteractionManager.runAfterInteractions(() => {
-            if (!textInputRef.current) {
-                return;
-            }
-
-            if (!shouldDelay) {
-                textInputRef.current.focus();
-            } else {
-                // Keyboard is not opened after Emoji Picker is closed
-                // SetTimeout is used as a workaround
-                // https://github.com/react-native-modal/react-native-modal/issues/114
-                // We carefully choose a delay. 100ms is found enough for keyboard to open.
-                setTimeout(() => textInputRef.current.focus(), 100);
-            }
-        });
+        focusWithDelay(textInputRef.current)(shouldDelay);
```

Comment:
can the `textInputRef` or `textInputRef.current` be null as we saw here? https://github.com/Expensify/App/pull/25981

is that handled well by the `focusWithDelay` method?

Reply:
Yes, that's handled by the focusWithDelay method!